Utils/Dataflow/README

```diff
@@ -91,6 +91,30 @@ intended for making sure that information is consistent between ProdSys2 and
 ES. It gets a very basic set of metadata from ProdSys2, adds ES-related
 fields, and checks that it is present in ES rather than uploading it.
 
+================
+* Stages types *
+================
+There are three types of stages:
+
+1. Source stage - an initial stage in a dataflow pipeline that gets data
+from an external source (CDS, Rucio, AMI, etc.)
+
+2. Transform stage - a stage that makes a set of different operations
+with metadata forming messages according to a particular pattern (e.g. select
```

Comment:
...on metadata...

Reply:
This, in my opinion, is a part of the larger problem. Usually it is possible to say "perform operations with metadata", but in this case we have a construction "perform operations with metadata forming messages according to pattern", which can be interpreted in many ways (especially if a possibility of missing comma or two is considered), such as:
1. Perform operations, and in these operations metadata forms messages according to pattern.
2. Perform operations according to pattern, and in these operations metadata forms messages.
3. Perform operations with metadata and, by doing so, form messages according to pattern.
4. Perform operations with some of the metadata - the ones that form messages according to pattern.
I suggest rewording the sentence to avoid such ambiguity, and changing "with" to "on" looks like a good start, as it "cuts off" some "branches".

Reply:
New variant uses the `operations on metadata` version so I guess this discussion may be resolverd now.